app/renderer/components/tabs/content/newSessionIcon.js

```diff
@@ -73,21 +64,17 @@ class NewSessionIcon extends React.Component {
 module.exports = ReduxComponent.connect(NewSessionIcon)
 
 const styles = StyleSheet.create({
-  icon: {
+  newSession__icon: {
+    zIndex: 99,
```

Comment:
Would you mind converting this into const and adding it to global.js so that we could see the relationship among z-index values?

Reply:
Confirmed this was addressed on https://github.com/brave/browser-laptop/pull/10691/files#diff-d8880b1566bd03fcd4e0b29425a5d473R70

On the commit: https://github.com/brave/browser-laptop/commit/049f438503ceab704cd9064b4a6811fea1debfda#diff-d8880b1566bd03fcd4e0b29425a5d473R70